### ISSUE TITLE
MIT License?

### ISSUE THREAD

**Issue by evanmrose** [lpiepiora/bower-webpack-plugin#37]
Is this plugin under the MIT license? If so, can you add the license file? Happy to add it and submit a PR @lpiepiora

**Comment by lpiepiora** [comment]
Yes it is. If you feel like you need a license file attached, feel free to submit a PR.
